exasol_advanced_analytics_framework/udf_communication/peer_communicator/frontend_peer_state.py

```diff
@@ -78,3 +78,7 @@ def received_connection_is_closed(self):
     @property
     def connection_is_closed(self) -> bool:
         return self._connection_is_closed
+
+    def received_acknowledge_payload_message(self, acknowledge_payload: messages.AcknowledgePayload):
+        """ Not yet implemented and for that reason we ignore the input"""
+        pass
```

Comment:
I learned from Nico that if you have a docstring you don't even need to declare "pass" here :)

Reply:
The pass first, yeah otherwise you are right, I forgot it.